src/PublicacaoService.java

```diff
@@ -81,4 +87,28 @@ private boolean comentarioPodeSerDeletado(Usuario usuario, Comentario comentario
                 comentario.getUsuario().getIdentificador() == usuario.getIdentificador() ||
                 usuario.getTipoUsuario() == Usuario.TipoUsuario.ADM;
     }
+
+    public void salvaPostagensArqCsv(String arq, Usuario usuario){
+        Path path = Paths.get(arq + ".csv");
+        List<Postagem> postagens = new ArrayList<Postagem>();
+        postagens = buscaPostagensPorUsuario(usuario);
+        try (PrintWriter writer = new PrintWriter(Files.newBufferedWriter(path, Charset.defaultCharset()))){
+
+            for (Postagem p: postagens){
+                writer.print(p.getTexto()+ ";" + p.getTags() + ";" + p.getLink());
```

Comment:
é csv...
ideal é ter um print antes do for declarando os headers.
writer.print("text;tags;link")